dsc/docs-conceptual/dsc-3.0/reference/schemas/outputs/resource/list.md

```diff
@@ -65,6 +77,62 @@ Required: true
 Pattern:  ^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
 ```
 
+### capabilities
+
+Defines the operations and behaviors the resource is implemented to support. This property is an
+array of capabilities. Resources always have the `Get` capability, but the other capabilities are
+optional and depend on the resource.
+
+The following list describes the available capabilities for a resource:
+
+- <a id="capability-get" /> `Get` - The resource supports retrieving the current state of an
+  instance. All DSC Resources must have this capability. A resource has this capability when it
+  defines the mandatory [get][07] property in its resource manifest.
+- <a id="capability-set" /> `Set` - The resource supports enforcing the desired state of an
+  instance. A resource has this capability when it defines the [set][08] property in its resource
+  manifest. Resources without this capability can't be used with the [dsc resource set][09] or
+  [dsc config set][10] commands unless they're in a [Microsoft.DSC/Assertion][11] group as a nested
```

Comment:
Can't figure out what `[Microsoft.DSC/Assertion][11]` is supposed to point to. The current definition is 404.

Reply:
```suggestion
  [dsc config set][10] commands unless they're in a `Microsoft.DSC/Assertion` group as a nested
```